Amplitude/AMPUtils.m

```diff
@@ -141,4 +141,13 @@ + (NSDictionary *) validateGroups:(NSDictionary *) obj
     return [NSDictionary dictionaryWithDictionary:dict];
 }
 
++ (NSString*) platformDataDirectory
+{
+#if TARGET_OS_TV
+    return [NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES) objectAtIndex: 0];
```

Comment:
Unfortunately it seems there's no persistent storage for tvOS apps, so we need to upload events immediately